stack-packs/packs/drupal.js

```diff
@@ -0,0 +1,71 @@
+/**
```

Comment:
can delete

Reply:
ah, right those files were meant to be deleted, forgot to earlier: #11396

Reply:
Sure. Deleted.